File: go/providers/generator_email.go
package providers

// GeneratorEmail is the generator.email provider (same gasmurl backend as emailfake).
type GeneratorEmail struct {
	*gasmurlBackend
}

// NewGeneratorEmail creates a new GeneratorEmail provider.
func NewGeneratorEmail(config map[string]string) *GeneratorEmail {
	return &GeneratorEmail{newGasmurlBackend(config, "generator.email", "https://generator.email", ".generator.email")}
}

File: javascript/src/providers/GeneratorEmail.js
import { Emailfake } from './Emailfake.js';

/**
 * generator.email provider — same gasmurl backend as emailfake.com.
 * Only the base URL and provider name differ; all logic is inherited.
 */
export class GeneratorEmail extends Emailfake {
  /** @param {{ randomUA?: boolean, useCookies?: boolean }|undefined} opts */
  constructor(opts = {}) {
    super(opts, 'generator.email', 'https://generator.email');
  }
}
